crates/fuel-core/src/service/adapters.rs

```diff
@@ -33,6 +37,24 @@ impl TxPoolAdapter {
     }
 }
 
+#[derive(Clone)]
+pub struct TransactionsSource {
+    txpool: TxPoolSharedState<P2PAdapter, Database>,
+    _block_height: BlockHeight,
+}
+
+impl TransactionsSource {
+    pub fn new(
+        txpool: TxPoolSharedState<P2PAdapter, Database>,
+        block_height: BlockHeight,
+    ) -> Self {
+        Self {
+            txpool,
+            _block_height: block_height,
```

Comment:
Looks like block_height isnt used, is the idea for adding it here to enable selecting txs based on maturity requirements?

Reply:
Yes:) The code before did the same
![IMG_1069](https://github.com/FuelLabs/fuel-core/assets/18346821/b028b71e-cb47-48f5-a823-9f476092eeed)